qualtrics_link/util.py

```diff
@@ -179,41 +184,131 @@ def is_user_in_whitelist(huid):
             return True
     except QualtricsAccessList.DoesNotExist:
         # the user is not in the whitelist'
+        logger.info('The HUID {} does not exist within the Qualtrics access list.'.format(huid))
         return False
 
 
-def build_user_dict(data):
+# If there are more than one record returned for a HUID,
+# we want to determine if any are employee status and to return that record.
+def filter_person_list(person_list):
+    for person in person_list:
+        if person.role_type_cd.lower() == 'employee':
+            return person
+    # If no employee records were found, then return the first person in the list
+    return person_list[0]
 
-    user_data = {
-        'role': 'generic',
-        'division': 'Other',
-        'validschool': False,
-        'validdept': False
+
+# Get the person list matching the given HUID or return None
+def get_person_list(huid):
+    person_list = Person.objects.filter(univ_id=huid, prime_role_indicator='Y')
```

Comment:
I'm a little concerned about limiting to `prime_role_indicator='Y'` here -- that data is pretty dirty and I worry we might miss people who don't have this flag on any of their roles. We'd definitely want to _prefer_ roles that are flagged as the prime role, though.